.circleci/config.yml

```diff
@@ -61,9 +63,8 @@ jobs:
             - deploy:
                 name: Deploy
                 command: |
-                    if [ "${CIRCLE_BRANCH}" = "development" ] && [ -n "$DEPLOY_HOST" ] && [ -n "$DEPLOY_USER" ] && [ -n "$DEPLOY_PASSWORD" ]; then
-                        sudo npm install -g grunt-cli
-                        grunt deploy --git-commit=$CIRCLE_SHA1 --ftp-host=$DEPLOY_HOST --ftp-user=$DEPLOY_USER --ftp-pass=$DEPLOY_PASSWORD
+                    if [ -n "$DEPLOY_HOST" ] && [ -n "$DEPLOY_USER" ] && [ -n "$DEPLOY_PASSWORD" ]; then
```

Comment:
Removing the condition "${CIRCLE_BRANCH}" = "development" would result in having builds from any branch deployed via ftp. This is in most cases not desired. Only from one agreed on branch deployments should happen. 

Btw, sorry, i owe you all some docu/howto on circle ci config ;)

Reply:
This was done temporarily to check ftp push feature in this branch .
This will restore before being merged

Reply:
ok, i'll keep this one open so we do not forget..